SocialCareCaseViewerApi.Tests/V1/Gateways/DatabaseGateway.Tests.cs

```diff
@@ -563,48 +563,83 @@ private Team SaveTeamToDatabase(Team team)
         [Test]
         public void GetWarningNotesReturnsTheExpectedWarningNote()
         {
-            WarningNote warningNote = new WarningNote()
+            var testPersonId = _faker.Random.Int();
+            var differentPersonId = _faker.Random.Int();
+
+            var warningNote = new WarningNote
             {
-                PersonId = 12345
+                PersonId = testPersonId
             };
-            WarningNote wrongWarningNote = new WarningNote()
+
+            var wrongWarningNote = new WarningNote
             {
-                PersonId = 67890
+                PersonId = differentPersonId
             };
+
             DatabaseContext.WarningNotes.Add(warningNote);
             DatabaseContext.WarningNotes.Add(wrongWarningNote);
             DatabaseContext.SaveChanges();
 
-            var request = _fixture.Build<GetWarningNoteRequest>()
-                            .With(x => x.PersonId, warningNote.PersonId)
-                            .Create();
-
-            var response = _classUnderTest.GetWarningNotes(request);
+            var response = _classUnderTest.GetWarningNotes(testPersonId);
 
             response.Should().ContainSingle();
             response.Should().ContainEquivalentOf(warningNote);
             response.Should().NotContain(wrongWarningNote);
         }
 
         [Test]
-        public void GetWarningNotesReturnsAnExceptionIfTheWarningNoteDoesNotExist()
+        public void GetWarningNotesReturnsAListOfWarningNotesForASpecificPerson()
         {
-            WarningNote warningNote = new WarningNote()
+            var testPersonId = _faker.Random.Int();
```

Comment:
This could be tider if we used TestHelpers.CreatePerson() and maybe created a function in TestHelpers for Creating Warning Notes (that can take a personId as a potential paramter)

Could also use Helper functions in test above.

Reply:
I agree 👍🏾 . I think we can do that in a separate PR.